src/sass/edu-benefits.scss

```diff
@@ -106,12 +107,22 @@ ul.claim-list{
   }
 }
 
+// Align with navigation buttons
+.omb-info--container {
```

Comment:
Are these edu only styles? Or something that would be necessary if we used this component elsewhere?